framework/src/media/MediaPlayerImpl.cpp

```diff
@@ -366,6 +366,11 @@ void MediaPlayerImpl::pausePlayer()
 	mCurState = PLAYER_STATE_PAUSED;
 }
 
+int MediaPlayerImpl::getId()
+{
+	return mId;
```

Comment:
Any possibility of an integer overflow?

Reply:
@davidfather yes, you right.
so, our framework will delete id.
could you check the pr : https://github.com/Samsung/TizenRT/pull/2034

Reply:
@junmin-kim do not merge!!